sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -105,9 +106,11 @@ def submit(self, proto_pipeline):
     Submit and run the pipeline defined by `proto_pipeline`.
     """
     prepare_response = self.prepare(proto_pipeline)
+    artifact_endpoint = (self.artifact_endpoint
```

Comment:
You will need to correct the formatting here:

```
12:35:40 -    artifact_endpoint = (self.artifact_endpoint
12:35:40 -                         or prepare_response.artifact_staging_endpoint.url)
12:35:40 +    artifact_endpoint = (
12:35:40 +        self.artifact_endpoint or
12:35:40 +        prepare_response.artifact_staging_endpoint.url)
```

Reply:
Ah thanks, sorry forgot to run the lint